provision/acc_provision/acc_provision.py

```diff
@@ -1508,6 +1518,12 @@ def provision(args, apic_file, no_random):
         err("overlapping subnets found in configuration input file")
         return False
 
+    # Verify that image_pull_secret is a valid K8s secret name and not a YAML string
+    if "registry" in config.keys() and "image_pull_secret" in config["registry"]:
+        if not check_image_pull_secret(config):
+            err("The image_pull_secret is Invalid. It is supposed to be name of the k8s secret and it shouldn't have characters like =,+,!,~,@,& etc")
```

Comment:
Rephrase this to say:
Invalid image_pull_secret value, it must be a valid DNS subdomain name.

Reply:
Yes updated as requested. Thanks for the input @snaiksat